workflows/RSV_illumina_pe_summary.wdl

```diff
@@ -17,156 +18,39 @@ workflow RSV_illumina_pe_summary {
         File concat_seq_results_py
```

Comment:
Should this be concatenate_seq_results_py or maybe should we shorten it everwhere?